Firestore/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- [fixed] Fixed a bug that caused the client to crash when path names with
```

Comment:
This is wordier than it needs to be and also doesn't quite capture the essence of the fix (since all characters are unicode characters). Non-ASCII would capture that precisely but is probably too technical.

Perhaps:

```
Fixed a crash when using path names with international characters with persistence enabled.
```

?

Reply:
sgtm; done.